pylint/testutils/configuration_test.py

```diff
@@ -31,7 +38,7 @@ def get_expected_or_default(
         with open(expected_result_path, encoding="utf8") as f:
             expected = f.read()
         # logging is helpful to realize your file is not taken into
-        # account after a misspell of the file name. The output of the
+        # account after a misspelling of the file name. The output of the
```

Comment:
```suggestion
        # account after a misspelling of the file name. The output of the
```

😄

Reply:
I don't know if it's the joke, but your suggestion does not change anything ? 😄

Reply:
Oh sorry! I wanted to add a 😄 for the fact that misspell was misspelled